lib/src/command/add.dart

```diff
@@ -563,6 +564,9 @@ Specify multiple sdk packages with descriptors.''');
             {
               'dependencies': {
                 packageName: parsedDescriptor,
+              },
+              'environment': {
+                'sdk': sdk.version.toString(),
```

Comment:
This is an entirely unrelated fix, right?

Reply:
Well - it was discovered because the suggestion of
```
dart pub add bar:'{"version":"^2.0.0","hosted":"http://localhost:44091"}
```
would otherwise fail because it uses a new-style hosted url...